practice/lib/windows/windows_actions.rb

```diff
@@ -0,0 +1,5 @@
+module Windows
+  def self.reboot_machine
+    system('shutdown /r /t 5')
```

Comment:
It would be nice to get seconds before reboot as an argument, like reboot_machine(seconds)
Then  we will have system("shutdown /r /t #{seconds}")
It allows use this method with different time before reboot

Reply:
@SDaniV Thanks, I have doubts, what you exactly want to see here